components/raftstore/src/store/snap.rs

```diff
@@ -1956,7 +1968,12 @@ impl TabletSnapManager {
     }
 
     pub fn get_tablet_checkpointer_path(&self, key: &TabletSnapKey) -> PathBuf {
-        let prefix = format!("{}_{}", SNAP_GEN_PREFIX, key);
+        let prefix = key.get_gen_suffix();
+        PathBuf::from(&self.base).join(prefix)
```

Comment:
What's wrong with the original implementation?

Reply:
@BusyJay I think both way are fine. In fact the new way is better IMO as it potentially eliminates the duplicate code of "format!("{}_{}", SNAP_GEN_PREFIX, key)"  in case somewhere else needs the path.

Reply:
fixed， I has remove the extra modify.